backend/src/main/java/com/htecgroup/skynest/service/impl/FileServiceImpl.java

```diff
@@ -101,6 +103,26 @@ public FileDownloadResponse downloadFile(UUID fileId) {
         fileMetadataEntity.getName(), fileMetadataEntity.getType(), fileContents);
   }
 
+  @Override
+  public List<FileResponse> getAllFilesFromRoot(UUID bucketId) {
+    List<FileMetadataEntity> allFiles =
+        fileMetadataRepository.findAllByBucketIdAndParentFolderIsNull(bucketId);
+    return getFileResponses(allFiles);
+  }
+
+  @Override
+  public List<FileResponse> getAllFilesFromParent(UUID parentFolderId) {
```

Comment:
Aren't those files at same level?

Reply:
Yes but that level is determined by the parent folder. We are returning first level of that particular folder sent by parameter.

Reply:
I renamed it also, I wont use it in this PR but in getFolderContent later it will be used so I added it now. Same for Folders.